File: src/models/features-sheet.ts

export class Topic {
    constructor(
        public topic: string,
        public order: number,
        public level: number,
        public title: string,
        public details: string,
        public notYetAvailable: boolean,
        public allowISO: string,
        public denyISO: string
    ) {}
}

export class FeaturesSheetLine {
    constructor(
        public topic: string
    ) {}
    public included: boolean;
    public standard: string;
    public premium: string;
    public professional: string;
    public enterprise: string;
}

export class FeaturesSheet {
    constructor(
        public name: string
    ) {}
    public lines: FeaturesSheetLine[] = [];
}

File: Models/FeaturesSheet.cs
 namespace Configurator.Models
 {
    public class FeaturesSheetLine
    {
        public string Topic { get; set; }
        public bool Included {get; set;}
        public string Standard {get; set;}
        public string Premium {get; set;}
        public string Professional {get; set;}
        public string Enterprise {get; set;}
    }

    public class FeaturesSheet
    {
        public string Name { get; set; }
        public FeaturesSheetLine[] Lines { get; set; }
    }
 }